src/index.html

```diff
@@ -228,7 +228,7 @@ <h3 class="padding-bottom-large">Data requirements</h3>
                         <ol>
                             <li>The genome assembly used is International Nucleotide Sequence Database Collaboration (<a href="http://www.insdc.org/" target="_blank">INSDC</a>) registered, or will be at point of submission</li>
                             <li>The variation data is described in valid VCF file(s) this can be tested prior to submission using the EVA VCF Validation Suite found <a href="https://github.com/EBIvariation/vcf-validator" target="_blank">here</a></li>
-                            <li>Submitted variants are associated with allele frequency values and/or genotypes and/or the raw materials to calculate allele frequency values internally (i.e. Allele Count AND Allele Number values)</li>
+                            <li>For all data submitted to the EVA, we require that it be possible to compute allele frequencies for all submitted variants. Therefore, the EVA supports two types of submissions 1) variation data with sample genotypes 2) summary data with population allele frequencies</li>

```

Comment:
I think this should have more punctuation (colon right before `1)`) or be split in 2 lines.